Templates/index.md

```diff
@@ -0,0 +1,32 @@
+# How to Use EOS Doc Templates
+
+To submit your documentation to the EOS Network Foundation (ENF), create a fork of the receiving repository and then add work on your own fork. From there, do a PR from your fork to the main branch. DevRel then reviews the PR, suggests changes, if any, and after approval the PR can be merged. 
+
+ENF Templates use MarkDown language.
+
+Templates provide a structure for your content.
+
+Text identified within brackets [...] are instructions to you for the type of information to include in the topic.
+
+If a section title contains "(Optional)" and you don’t need to add information for that topic, add the description “N/A”.
```

Comment:
Not sure why these are separate paragraphs. They are all properties of ENF templates. Why not converting these to bullets and adding a stem sentence above, something like:

"The ENF Doc Templates are a guide for contributors to submit content to the EOS Documentation Portal. The Doc Templates provide the following:"

Reply:
Done